tidy3d/components/data/dataset.py

```diff
@@ -354,31 +355,38 @@ class ModeSolverDataset(ElectromagneticFieldDataset):
         description="Complex-valued effective propagation constants associated with the mode.",
     )
 
-    n_group_raw: ModeIndexDataArray = pd.Field(
+    n_group_raw: GroupIndexDataArray = pd.Field(
         None,
-        alias="n_group",
+        alias="n_group",  # This is for backwards compatibility only when loading old data
         title="Group Index",
         description="Index associated with group velocity of the mode.",
     )
 
+    dispersion_raw: ModeDispersionDataArray = pd.Field(
+        None,
+        title="Dispersion",
+        description="Dispersion parameter for the mode.",
+        units=PICOSECOND_PER_NANOMETER_PER_KILOMETER,
```

Comment:
This is a crazy unit lol, is this standard?

Reply:
Yes... https://www.rp-photonics.com/group_velocity_dispersion.html

Reply:
Oh I see I guess it makes sense if you think about long-range fiber optics.